<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>791</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string>Mesh and material properties</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1" colspan="2">
      <widget class="QGroupBox" name="groupBox_7">
       <property name="title">
        <string>Clamps</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Clamps stiffness :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="stiffnessX">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLineEdit" name="stiffnessY">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLineEdit" name="stiffnessZ">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>[kN/mm]</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Clamps damping :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="dampingX">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLineEdit" name="dampingY">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLineEdit" name="dampingZ">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>[Ns/mm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QGroupBox" name="groupBox_8">
       <property name="title">
        <string>Ballast</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Select ballast mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ballastMeshes"/>
        </item>
        <item>
         <widget class="QPushButton" name="newBallastMeshButton">
          <property name="text">
           <string>Browse new ballast mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Ballast material</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ballastMaterial"/>
        </item>
        <item>
         <widget class="QPushButton" name="newBallastMaterialButton">
          <property name="text">
           <string>Browse new ballast material</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Rails</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Select rail mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="railMeshes"/>
        </item>
        <item>
         <widget class="QPushButton" name="newRailMeshButton">
          <property name="text">
           <string>Browse new rail mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Rail material</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="railMaterial"/>
        </item>
        <item>
         <widget class="QPushButton" name="newRailMaterialButton">
          <property name="text">
           <string>Browse new rail material</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>USPs</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="cb_USP">
          <property name="text">
           <string>Include USPs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Select USP mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="USPMeshes">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="newUSPMeshButton">
          <property name="text">
           <string>Browse new USP mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>USP material</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="USPMaterial"/>
        </item>
        <item>
         <widget class="QPushButton" name="newUSPMaterialButton">
          <property name="text">
           <string>Browse new USP material</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Pads</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Select pad mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="padMeshes"/>
        </item>
        <item>
         <widget class="QPushButton" name="newPadMeshButton">
          <property name="text">
           <string>Browse new pad mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Pad material 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="padMaterial1"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Pad material 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="padMaterial2"/>
        </item>
        <item>
         <widget class="QPushButton" name="newPadMaterialButton">
          <property name="text">
           <string>Browse new pad material</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Sleepers</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Select sleeper mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sleeperMeshes"/>
        </item>
        <item>
         <widget class="QPushButton" name="newSleeperMeshButton">
          <property name="text">
           <string>Browse new sleeper mesh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Sleeper material</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sleeperMaterial"/>
        </item>
        <item>
         <widget class="QPushButton" name="newSleeperMaterialButton">
          <property name="text">
           <string>Browse new sleeper material</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Sleeper spacing [mm]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="slpDist">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>600</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QGroupBox" name="groupBox_9">
       <property name="title">
        <string>Actions</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="btn_clear">
          <property name="text">
           <string>Clear folders / Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_updateCB">
          <property name="text">
           <string>Update drop-down menus</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="runButton">
          <property name="text">
           <string>Run Simulation</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>791</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Study details</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name of the simulation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameSimu"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Saving location (Absolute path)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pathWorkingDirectory"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
